### DISCUSSION TITLE
When doing a vertical split of current buffer to the right

### DISCUSSION THREAD

**Discussion by bcardarella** [zed-industries/zed#12132] [Feature Requests]
If I split the current buffer vertically with `:vs` the result is my original pane with the other open files is now on the right and the new split is on the left. Can I reverse this so newly split panes open to the right and my current pane with other files stays on the left?

**Comment by baldwindavid** [comment]
I'm not aware of a way to configure the split setting nor a way to override it via keymap. There is the split right command you've probably seen...

![Screenshot 2024-05-22 at 14 28 08](https://github.com/zed-industries/zed/assets/4578/6e5a83c7-8b26-441d-b5db-6437d68004ea)

It never bothered me because I use a keybinding for splitting, but imagine that's tough if you've got years of muscle-memory for `:vs`.

```json
"space \\": "pane::SplitRight",
```

**Comment by MiguelCiulog** [comment]
you can use `"pane_split_direction_vertical": "right"` on your settings.json

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
